packages/language/src/validation/messages/IBM1295IE-sole-bound-specified.ts

```diff
@@ -18,17 +18,17 @@ export function IBM1295IE_sole_bound_specified(
   bound: DimensionBound,
   accept: PliValidationAcceptor,
 ): void {
-  if (bound.bound2 !== undefined) {
+  if (bound.lower !== undefined) {
```

Comment:
Just to be sure, `bound2` is supposed to be `lower`?

Reply:
Not really, the old logic was really complicated. Essentially, the parser parses the bounds as `${bound1}:${bound2}`, with the `:${bound2}` being optional. If `bound2` is specified, it becomes the upper bound. If it's not specified `bound1` is the upper bound. In Langium, we were unable to represent this logic in the AST (which is why we used `bound1`, `bound2` as names in the first place).

For this case: `lower` is only set, in case we actually parse the `${bound1}:${bound2}` construct. In this case, we no longer have just a sole bound. Therefore, the validation does not apply anymore and we return.

Reply:
Makes sense, thanks for the clarification